scripts/backup/monitor

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+check_root () {
+  if [[ $UID != 0 ]]; then
+    echo "Error: This script must be run as root."
```

Comment:
We should probably go through the `karen` triggers and drop privs to the UID 1000 user for any trigger that doesn't require root.

Reply:
True, although we probably need this trigger to run as root because we need to copy `db/user.json` which is created by root since `umbrel-manager`, the service container that creates it, runs as root.

We can drop root requirements once all the dockerized containers run rootless with UID 1000.